Fragments/Filter/AtLeastOneLepton.dat

```diff
@@ -5,4 +5,4 @@ lheGenericFilter = cms.EDFilter("LHEGenericFilter",
     AcceptLogic = cms.string("GT") 
 )
 
-ProductionFilterSequence = cms.Sequence(lheGenericFilter+generator)
\ No newline at end of file
+ProductionFilterSequence = cms.Sequence(lheGenericFilter+generator)
```

Comment:
Was this just a mistake commit? I don't see the line changes but just to confirm